index.html

```diff
@@ -84,7 +84,7 @@ <h1>Safari Trip Advisory</h1>
 	
 			<!--About_Us -->
 		
-<div id="About_us">
+<div class="About_us">
 		<div class="about_us_A">
 			<img src="images/logo.png" alt="logo">
 		</div>	

```

Comment:
V7 medicine and clothes & toiletries icons were selected while other icons were selected from Version 8... for the what to bring section